camel-idea-plugin/src/main/java/org/apache/camel/idea/CamelProjectComponent.java

```diff
@@ -79,7 +79,8 @@ public void rootsChanged(ModuleRootEvent event) {
                     getCamelIdeaService(project).clearLibraries();
 
                     for (Module module : ModuleManager.getInstance(project).getModules()) {
-                        getCamelIdeaService(project).scanForCamelDependencies(module);
+                        getCamelIdeaService(project).scanForCamelVersionChange(project, module);
```

Comment:
We need to separate them because we want to detect the camel version first. Its easier and safter code when its separated.